sccm/core/servers/deploy/configure/azure-services-wizard.md

```diff
@@ -209,6 +209,12 @@ Select **Sign in** to authenticate to Azure as an administrative user. These cre
 
 Select **OK** to create the native app in Azure AD and close the Create Client Application dialog. This action returns to the [Client App dialog](#client-app-dialog).
 
+### Renew secret key Azure AD apps
+Prior to 1806 version, to renew secret key of an Azure App, you need to recreate the App.
```

Comment:
A couple of minor issues on this line:
- Add "the" before "secret key": "...to renew the secret key..."
- lowercase "app" (both instances)

```suggestion
Prior to 1806 version, to renew the secret key of an Azure app, you need to recreate the app.
```